### ISSUE TITLE
HTTP / HTTPS Awareness

### ISSUE THREAD

**Issue by jakraska** [avatarnewyork/dockerenv_apache#5]
PHP / Apache needs to be aware about whether the site is in HTTPS or HTTP.   This is often used in `.htaccess` files and sometimes in PHP (the secure pages drupal module)

Typically you can detect https in a .htaccess file with the variable `%{HTTPS}`

Example:

```
RewriteCond %{HTTPS} =on
```

For some reason, in the past on our dev environments that didnt work (not sure if it was because of something specific, an old setup or what).  So in additon to that, in all https vhosts on gil we have an environmental variable `ENV:HTTPS` that we check.

In short, basically i need either one of these to be correctly set when accessing the site via https so that these rewrite rules will work:

```
RewriteCond %{ENV:HTTPS} =on [OR]
RewriteCond %{HTTPS} =on
```
